openapi/diff/action.yml

```
@@ -0,0 +1,74 @@
+name: OpenAPI Diff
+description: Generate a diff between 2 OpenAPI specifications
+
+inputs:
+  base:
+    description: Base diff specification file
+    required: true
+  head:
+    description: Head diff specification file
+    required: true
+  output:
+    description: Output directory
+    required: true
+    default: reports/openapi
+
+outputs:
+  markdown:
+    description: Path to the generated markdown diff file
+    value: ${{ steps.generate.outputs.markdown }}
+  text:
+    description: Path to the generated text diff file
+    value: ${{ steps.generate.outputs.text }}
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Use Node.js
+      uses: actions/setup-node@v4
+
+    - name: Install bump CLI
+      run: |
+        npm install -g bump-cli
+      shell: bash
+
+    - name: Create outputs directory
+      run: |
+        mkdir -p ${{ inputs.output }}
+      shell: bash
+
+    - name: generate markdown diff
```

As it will generate both markdown and text diffs

```suggestion
    - name: generate diff files (markdown & plain text)
```